### PR TITLE
BUG Fixed setting LastEdited for DataObject with class ancestry

### LINKED ISSUE #4207: [Bug] Testing: failed to set LastEdited field from fixture
Test case 1:

Setting LastEdited field on simple DataObject from fixture results in DB error.
(object ID is missing in where clause)

Test case 2:

Setting LastEdited field on DataObject with class ancestry results in DB error.

For example for Page <|-- SiteTree, the code tries to set Page.LastEdited field.

### PR DESCRIPTION
**Pull request by gregsmirnov**
fixes #4207

### CODE REVIEW

File: dev/FixtureBlueprint.php [hunk 1]
```diff
@@ -183,14 +183,8 @@ public function createObject($identifier, $data = null, $fixtures = null) {
 
 			// If LastEdited was set in the fixture, set it here
 			if($data && array_key_exists('LastEdited', $data)) {
-				$edited = $this->parseValue($data['LastEdited'], $fixtures);
```
Comment (tractorcow) [inline dev/FixtureBlueprint.php]: Would it be simpler to simply do `$class = ClassInfo::baseDataClass($class);`?

Comment (tractorcow) [inline dev/FixtureBlueprint.php]: Or if you want to do it generically `$class = ClassInfo::table_for_object_field($class, 'LastEdited');`

**Comment by tractorcow** [comment]
Thanks for the fix @gregsmirnov. I just gave a few suggestions that might simplify the code.

**Comment by gregsmirnov** [comment]
@tractorcow Thank you for comments. Replacing this pull request with pull #4220

**Pull request closed without merging**